platform/darwin/src/MGLNetworkConfiguration.m

```diff
@@ -65,8 +67,9 @@ - (NSURLSessionConfiguration *)defaultSessionConfiguration {
 }
 
 - (void)startDownloadEvent:(NSString *)urlString type:(NSString *)resourceType {
-    if (urlString && ![self.events objectForKey:urlString]) {
-        [self.events setObject:@{ MGLStartTime: [NSDate date], MGLResourceType: resourceType } forKey:urlString];
+    NSDate *startDate = [NSDate date];
```

Comment:
Nit: move this declaration into the following `if` statement.

Reply:
I had this outside to try and give a more accurate time, but the end-date isn't, so 🤷‍♂. I'm going to move this inside - though we should think about using `CACurrentMediaTime` - using `NSDate` here is a little overkill.